pytripgui/plan_executor/html_executor_logger.py

```diff
@@ -0,0 +1,43 @@
+
+from queue import Queue
+from pytrip.tripexecuter import ExecutorLogger
+
+
+class HtmlExecutorLogger(ExecutorLogger):
+    def __init__(self):
+        self.queue = Queue()
+
+    def info(self, text):
+        self.queue.put("<b>{}</b>".format(text))
+
+    def log(self, text):
+        text = self._format_tags(text)
+        text = self._format_ansi(text)
+        text = self._format_colors(text)
```

Comment:
What about chaining calls into single line?

Reply:
@rozchlastywacz Would you mind to present the proposed solution in the form of a code?

Reply:
@ljelen Something like that:
`text = self._format_colors(self._format_ansi(self._format_tags(text)))`

Reply:
In my opinion, the unfolded version is more readable


P.S.
i'm thinking about other notation:
text = text._format_tags()._format_ansi()._format_colors()
or different one, but it requires more work (i.e providing another class)